tests (CI): faucet#TestFacebook: no ethereum address found to fund

This test has been seen initially by @ziogaschr failing before on Travis, too, during development.
Neither of us can reproduce it locally. 

https://travis-ci.org/github/etclabscore/core-geth/jobs/773517402
https://github.com/etclabscore/core-geth/commit/e9b5f6068ebfa4445fa66ad126d29b5ae9adaebb

```
INFO [06-04|17:39:43.045] Checking datadir migration               datadir=/tmp/faucet-migration-test-datadir-1622828383044933086/mychain

WARN [06-04|17:39:43.045] Found existing 'MultiFaucet' directory, migrating old=/tmp/faucet-migration-test-datadir-1622828383044933086/mychain/MultiFaucet new=/tmp/faucet-migration-test-datadir-1622828383044933086/mychain/CoreFaucet

--- FAIL: TestFacebook (0.92s)

    faucet_test.go:62: No Ethereum address found to fund

FAIL

FAIL	github.com/ethereum/go-ethereum/cmd/faucet	0.958s
```

--- a/cmd/faucet/faucet_test.go
+++ b/cmd/faucet/faucet_test.go
@@ -48,6 +48,8 @@ func TestMigrateFaucetDirectory(t *testing.T) {
 }
 
 func TestFacebook(t *testing.T) {
+	// TODO: Remove facebook auth or implement facebook api, which seems to require an API key
+	t.Skipf("The facebook access is flaky, needs to be reimplemented or removed")
 	for _, tt := range []struct {
 		url  string
 		want common.Address

